lib/lndclient/LndClient.ts

```diff
@@ -205,6 +205,32 @@ class LndClient extends BaseClient {
     return this.unaryCall<lndrpc.SendRequest, lndrpc.SendResponse.AsObject>('sendPaymentSync', request);
   }
 
+  /**
+   * Sends a payment over the Lightning Network.
+   * @param r_hash the payment hash to use for the payment
+   * @param amount the number of satoshis (or equivalent) to send
+   * @param destination the pub key of the recipient
+   * @param finalCltvDelta the CLTV delta from the current height for the time lock of the final hop (default: 9)
+   */
+  public sendPayment = (r_hash: Buffer | string, amount: number, destination: Buffer | string, finalCltvDelta = 9):
+  Promise<lndrpc.SendResponse.AsObject> => {
+    const request = new lndrpc.SendRequest();
+    if (typeof r_hash === 'string') {
+      request.setPaymentHashString(r_hash);
+    } else {
+      request.setPaymentHash(r_hash);
+    }
+    request.setAmt(amount);
```

Comment:
Is there really a good reason to replicate function signature and the function handling form LND?

Reply:
What do you mean?